test/integration2/util/knex-instance-provider.js

```diff
@@ -1,5 +1,6 @@
 const { promisify } = require('util');
 const knex = require('../../../lib');
+const { cloneDeep } = require('lodash');
```

Comment:
please import feature directly, `const cloneDeep = require('lodash/cloneDeep');`, so that it could be tree-shaken

Reply:
it's quite irrelevant for test files though